### PR TITLE
Docs: Update requirements for AMP script location

### PR DESCRIPTION
**Pull request by eddhannay**
See issue #612

- Remove requirement that AMP JS library is the last element in the head

### CODE REVIEW

File: spec/amp-html-format.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ AMP HTML documents MUST
 - <a name="canon"></a>contain a `<link rel="canonical" href="$SOME_URL" />` tag inside their head that points to the regular HTML version of the AMP HTML document or to itself if no such HTML version exists. [🔗](#canon)
 - <a name="chrs"></a>contain a `<meta charset="utf-8">` tag as the first child of their head tag. [🔗](#chrs)
 - <a name="vprt"></a>contain a `<meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">` tag inside their head tag. [🔗](#vprt)
-- <a name="scrpt"></a>contain a `<script async src="https://cdn.ampproject.org/v0.js"></script>` tag as the last element in their head. [🔗](#scrpt)
+- <a name="scrpt"></a>contain a `<script async src="https://cdn.ampproject.org/v0.js"></script>` tag inside their head. [🔗](#scrpt)
```
Comment (dvoytenko) [inline spec/amp-html-format.md]: "head tag"

**Comment by dvoytenko** [comment]
Thanks a lot! LGTM with one comment.

**Comment by eddhannay** [comment]
Added the word 'tag' as per @dvoytenko's comment

**Comment by dvoytenko** [comment]
Thanks a lot! LGTM. But could you rebase so I can merge?

**Pull request closed without merging**